zstacklib/zstacklib/gpu/vendors/kunlunxin.py

```
@@ -280,6 +274,24 @@ def get_basic_info(cls):
 
+    # ==========================================================================
+    # Addon Info Enrichment (productName)
+    # ==========================================================================
+
+    @classmethod
+    def enrich_addon_info(cls, gpu_info_map, pci_addresses):
+        """Add productName for Kunlunxin XPUs."""
+        if not pci_addresses:
+            return
+        from zstacklib.utils.gpu import get_kunlunxin_gpu_product_name_cmd, get_kunlunxin_product_name
+        r, o, e = bash_roe(get_kunlunxin_gpu_product_name_cmd())
```

Ruff (0.15.4)

[warning] Unpacked variable `e` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)